group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorService.java

```diff
@@ -0,0 +1,581 @@
+        }
+
+        return FutureUtils.failedFuture(Errors.UNSUPPORTED_VERSION.exception(
+            "This API is not implemented yet."
+        ));
+    }
+
+    /**
+     * See {@link GroupCoordinator#fetchAllOffsets(RequestContext, String, boolean)}.
+     */
+    @Override
+    public CompletableFuture<List<OffsetFetchResponseData.OffsetFetchResponseTopics>> fetchAllOffsets(
+        RequestContext context,
+        String groupId,
+        boolean requireStable
+    ) {
+        if (!isActive.get()) {
+            return FutureUtils.failedFuture(Errors.COORDINATOR_NOT_AVAILABLE.exception());
+        }
+
+        return FutureUtils.failedFuture(Errors.UNSUPPORTED_VERSION.exception(
+            "This API is not implemented yet."
+        ));
+    }
+
+    /**
+     * See {@link GroupCoordinator#commitOffsets(RequestContext, OffsetCommitRequestData, BufferSupplier)}.
+     */
+    @Override
+    public CompletableFuture<OffsetCommitResponseData> commitOffsets(
+        RequestContext context,
+        OffsetCommitRequestData request,
+        BufferSupplier bufferSupplier
+    ) {
+        if (!isActive.get()) {
+            return FutureUtils.failedFuture(Errors.COORDINATOR_NOT_AVAILABLE.exception());
+        }
+
+        return FutureUtils.failedFuture(Errors.UNSUPPORTED_VERSION.exception(
+            "This API is not implemented yet."
+        ));
+    }
+
+    /**
+     * See {@link GroupCoordinator#commitTransactionalOffsets(RequestContext, TxnOffsetCommitRequestData, BufferSupplier)}.
+     */
+    @Override
+    public CompletableFuture<TxnOffsetCommitResponseData> commitTransactionalOffsets(
+        RequestContext context,
+        TxnOffsetCommitRequestData request,
+        BufferSupplier bufferSupplier
+    ) {
+        if (!isActive.get()) {
+            return FutureUtils.failedFuture(Errors.COORDINATOR_NOT_AVAILABLE.exception());
+        }
+
+        return FutureUtils.failedFuture(Errors.UNSUPPORTED_VERSION.exception(
+            "This API is not implemented yet."
+        ));
+    }
+
+    /**
+     * See {@link GroupCoordinator#deleteOffsets(RequestContext, OffsetDeleteRequestData, BufferSupplier)}.
+     */
+    @Override
+    public CompletableFuture<OffsetDeleteResponseData> deleteOffsets(
+        RequestContext context,
+        OffsetDeleteRequestData request,
+        BufferSupplier bufferSupplier
+    ) {
+        if (!isActive.get()) {
+            return FutureUtils.failedFuture(Errors.COORDINATOR_NOT_AVAILABLE.exception());
+        }
+
+        return FutureUtils.failedFuture(Errors.UNSUPPORTED_VERSION.exception(
+            "This API is not implemented yet."
+        ));
+    }
+
+    /**
+     * See {@link GroupCoordinator#onTransactionCompleted(long, Iterable, TransactionResult)}.
+     */
+    @Override
+    public void onTransactionCompleted(
+        long producerId,
+        Iterable<TopicPartition> partitions,
+        TransactionResult transactionResult
+    ) {
+        throwIfNotActive();
```

Comment:
do we plan to add more to these?

Reply:
What do you mean? Different ones?

Reply:
Yeah, we have to implement of these methods. We have JIRAs for all of them...

Reply:
got it. thanks for clarifying. maybe we don't need a todo or anything if we at least have the jira.
Some of the other methods had "not yet implemented" in the body so I wasn't sure about the ones that didn't.